package/share-root.sh

```diff
@@ -3,10 +3,8 @@ set -x
 
 trap 'exit 0' SIGTERM
 ID=$(grep :devices: /proc/self/cgroup | head -n1 | awk -F/ '{print $NF}' | sed -e 's/docker-\(.*\)\.scope/\1/')
-IMAGE=$(docker inspect -f '{{.Config.Image}}' $ID)
 bash -c "$1"
 
-docker run --privileged --net host --pid host -v /:/host --rm --entrypoint /usr/bin/share-mnt $IMAGE "${@:2}" -- norun
 while ! docker start kubelet; do
```

Comment:
you might want to remove this restart kubelet also

Reply:
Well, this is a start call, so if it's already running it should be a no-op, but if it isn't then we start it (not sure why it would be stopped though)

Reply:
Oh guessing it would be failed before share-mount ran, after share-mount finishes then we can start it again. So I would agree, should be able to drop it.

Reply:
yeah, kubelet container is stopped because the mount is not shared. Once share-mnt is run, it tries to restart kubelet then. Without this logic we can safely drop it.